tests/providers/amazon/aws/hooks/test_emr.py

```diff
@@ -190,3 +190,100 @@ def test_get_cluster_id_by_name(self):
         no_match = hook.get_cluster_id_by_name("foo", ["RUNNING", "WAITING", "BOOTSTRAPPING"])
 
         assert no_match is None
+
+    @mock_emr
+    def test_send_cancel_steps_first_invocation(self):
+        """
+        Test that we can resolve cluster id by cluster name.
```

Comment:
This looks like copy/paste (same for the below method).

Reply:
Yup, updated the testcase names and docstring for each case according to the scenario it tests